### ISSUE TITLE
Accessibility improvements for default Identity UI

### ISSUE THREAD

**Issue by mkArtakMSFT** [dotnet/aspnetcore#33341]
The accessibility testing team has reported multiple issues with our default project template, related to the default identity UI:
- https://dev.azure.com/devdiv/DevDiv/_workitems/edit/1163091
- https://dev.azure.com/devdiv/DevDiv/_workitems/edit/1162814
- https://dev.azure.com/devdiv/DevDiv/_workitems/edit/1162783

**Comment by HaoK** [comment]
Note: this actually seems to come from bootstrap 4's default colors itself, should we point to this documentation?

https://getbootstrap.com/docs/4.0/getting-started/accessibility/

> Color contrast
Most colors that currently make up Bootstrap’s default palette—used throughout the framework for things such as button variations, alert variations, form validation indicators—lead to insufficient color contrast (below the recommended WCAG 2.0 color contrast ratio of 4.5:1) when used against a light background. Authors will need to manually modify/extend these default colors to ensure adequate color contrast ratios.

**Comment by HaoK** [comment]
Looks like all the contrast warnings are due to the selected text not having enough contrast, I've updated to the following (black/white):

Added to site.css
```
::-moz-selection {
    color: #fff;
    background-color: #333;
}
::selection {
    color: #fff;
    background-color: #333;
}
```

After: (Passes contrast ratio of 12.634
![image](https://user-images.githubusercontent.com/6537861/124671271-17ac4e80-de6a-11eb-8d73-28c14a6a5b50.png)

Before: (Fails contrast ratio of 3.065 < 4.5)
![image](https://user-images.githubusercontent.com/6537861/124671386-3ca0c180-de6a-11eb-9be7-475102f4e15e.png)

@blowdart @danroth27 are you guys okay with the new color choices here for selected text?  If not feel free to suggest something else and I can try the contrast

**Comment by blowdart** [comment]
If it passes I'm good with your selection.
